_test/test_multifit/test_multifit_horace_dnd_2.m

```diff
@@ -0,0 +1,90 @@
+classdef test_multifit_horace_dnd_2 < TestCaseWithSave
+    % Performs some tests of fitting to Horace objects using multifit_sqw_sqw
+    % Optionally writes results to output file
+    %
+    %   >> runtests test_multifit_horace_2  % Compares with previously saved results in
+    %                                       % test_multifit_horace_2_output.mat
+    %                                       % in the same folder as this function
+    %
+    %   >> test_multifit_horace_2('-save)   % Save to test_multifit_horace_2_output.mat
+    %                                       % in the Matlab temporary folder (copy to
+    %                                       % the same folder as this function after)
+    %
+    %   >> runtests test_multifit_horace_2:<test_func> % run a particular test from this
+    %                                       % this test suite
+    %
+    % Reads previously created test data sets.
+    
+    properties
+        w1data
+        w2data
+        win
+    end
+    
+    methods
+        function this=test_multifit_horace_dnd_2(name)
+            % Construct object
+            output_file = 'test_multifit_horace_2_dnd_output.mat';
+            this = this@TestCaseWithSave(name, output_file);
+            
+            % Read in data
+            data_dir = fileparts(mfilename('fullpath'));
+            
+            %CHANGED to work with new sqw class (read_ removed)
+            sqw_w1data = sqw(fullfile(data_dir,'w1data.sqw')); % was read_sqw
+            sqw_w2data = sqw(fullfile(data_dir,'w2data.sqw')); % was read_sqw
+            this.w1data = d1d(sqw_w1data); 
+            this.w2data = d1d(sqw_w2data); 
+            this.win=[this.w1data,this.w2data];     % combine the two cuts into an array of sqw objects and fit
+            
+            % Save output, if requrested
+            %this.save();
+        end
+        
+        % ------------------------------------------------------------------------------------------------
+        function this=test_fit(this)
```

Comment:
test fit does what??

this test looks to verify that the fore- and background functions can be reversed